src/transformers/models/qwen2_vl/modeling_qwen2_vl.py

```diff
@@ -286,9 +287,9 @@ def forward(
         q = apply_rotary_pos_emb_vision(q.unsqueeze(0), rotary_pos_emb).squeeze(0)
         k = apply_rotary_pos_emb_vision(k.unsqueeze(0), rotary_pos_emb).squeeze(0)
 
-        attention_mask = torch.zeros([1, seq_length, seq_length], device=q.device, dtype=torch.bool)
+        attention_mask = torch.full([1, seq_length, seq_length], float("-inf"), device=q.device, dtype=torch.float)
```

Comment:
the dtype should probably be the same as q.dtype. so 
```suggestion
        attention_mask = torch.full([1, seq_length, seq_length], torch.finfo(q.dtype).min, device=q.device, dtype=q.dtype)
```

Reply:
I have tried both method, still the same error is there any other place need to be modified?

Reply:
＋1

Reply:
+1

Reply:
> I have tried both method, still the same error is there any other place need to be modified?

Can you describe what kind of error it is?

Reply:
+1

Reply:
yep you need both dtype and `torch.finfo(q.dtype).min`